server/scripts/update-operators.sh

```diff
@@ -11,14 +12,18 @@ if [ ! -d "data/community-operators" ]; then
   fi
 
   git clone $OPERATORS_REPO community-operators || exit 1
-  popd
+fi
 
-  exit 0
+if [ ! $2 ]; then
+  echo "Using master branch"
+  OPERATORS_BRANCH=master
+else
+  OPERATORS_BRANCH=$2
 fi
```

Comment:
Maybe echo regardless so we can see if a non-master branch is pulled

Reply:
oh good catch